committeeoversightapp/templates/partials/compare_current_committees_table.html

```diff
@@ -0,0 +1,56 @@
+<div class="card mt-5 col-12 col-lg-10 mx-auto card-shadow">
+  <h2 class="text-center mt-4">{{title}}</h2>
+  <div class="card-body">
+    <div class="table-responsive">
+      <table class="committee-table table table-borderless table-striped table-sm text-center"
+      >
+      <thead>
+        <tr class="landing-row">
+          <td></td>
+          <td colspan="3" class="font-weight-bold">
+              Investigative Oversight Hearings
+              <hr class="mb-0" />
+          </td>
+          <td colspan="3" class="font-weight-bold">
+              Policy/Legislative Hearings
+              <hr class="mb-0" />
+          </td>
+          <td colspan="3" class="font-weight-bold">
+              Total Hearings
+              <hr class="mb-0" />
+          </td>
+        </tr>
+        <tr>
+          <th class="font-weight-normal small">Committee</th>
```

Comment:
A few of these headers occasionally appear misaligned with their sorting buttons, so you may want to add `align-top` here as well:

<img width="1016" alt="Screen Shot 2019-11-12 at 1 17 07 PM" src="https://user-images.githubusercontent.com/14170650/68703239-d73ee500-054f-11ea-81a5-7a0873ed3ade.png">